cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -348,7 +349,9 @@ pattern AsShelleyWitness :: AsType (KeyWitness ShelleyEra)
 pattern AsShelleyWitness = AsKeyWitness AsShelleyEra
 {-# COMPLETE AsShelleyWitness #-}
 
-
+-- We implement a custom serialization instance that differs from
+-- cardano-ledger because we want to be able to tell the difference between
+-- on disk witnesses for the cli's 'assemble' command.
```

Comment:
Suggest: `s/We implement a custom serialization instance that differs/This custom instance differs`